UI/window-basic-source-select.cpp

```diff
@@ -340,12 +346,10 @@ void OBSBasicSourceSelect::on_buttonBox_accepted()
 			OBSDataAutoRelease dat =
 				obs_data_create_from_json(data.c_str());
 			OBSSource source;
+			OBSSceneItem item;
 			AddNew(NULL, obs_data_get_string(dat, "id"),
 			       obs_data_get_string(dat, "name"),
-			       obs_data_get_bool(dat, "visible"), source);
-			OBSSceneItemAutoRelease item =
-				obs_scene_sceneitem_from_source(
-					main->GetCurrentScene(), source);
+			       obs_data_get_bool(dat, "visible"), source, item);
```

Comment:
I'd prefer using `!!` to be more explicit that this is a conversion to boolean.

Reply:
Changed.

Reply:
Or not item, but `source`. `source` is being used as a boolean for the `visible` parameter isn't it?

Reply:
Ah I misread due to the way Github shows the diff. My apologies.